From 3cf801248727fecd46186cb52821704ce4968d8c Mon Sep 17 00:00:00 2001
From: David Cohen <dacohen@gmail.com>
Date: Thu, 24 Feb 2011 12:51:32 -0800
Subject: [PATCH 096/258] omap: iommu: print module name on error messages

OMAP IOMMU generic layer doesn't need ot print function name during
error messages. Print module name instead which is more useful.

Signed-off-by: David Cohen <dacohen@gmail.com>
Signed-off-by: Tony Lindgren <tony@atomide.com>
---
 arch/arm/plat-omap/iommu.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/plat-omap/iommu.c b/arch/arm/plat-omap/iommu.c
index 3d8f55e..4b3218e 100644
--- a/arch/arm/plat-omap/iommu.c
+++ b/arch/arm/plat-omap/iommu.c
@@ -809,7 +809,7 @@ static irqreturn_t iommu_fault_handler(int irq, void *data)
 	iopgd = iopgd_offset(obj, da);
 
 	if (!iopgd_is_table(*iopgd)) {
-		dev_err(obj->dev, "%s: da:%08x pgd:%p *pgd:%08x\n", __func__,
+		dev_err(obj->dev, "%s: da:%08x pgd:%p *pgd:%08x\n", obj->name,
 			da, iopgd, *iopgd);
 		return IRQ_NONE;
 	}
@@ -817,7 +817,7 @@ static irqreturn_t iommu_fault_handler(int irq, void *data)
 	iopte = iopte_offset(iopgd, da);
 
 	dev_err(obj->dev, "%s: da:%08x pgd:%p *pgd:%08x pte:%p *pte:%08x\n",
-		__func__, da, iopgd, *iopgd, iopte, *iopte);
+		obj->name, da, iopgd, *iopgd, iopte, *iopte);
 
 	return IRQ_NONE;
 }
-- 
1.7.1

